presto-main/src/test/java/com/facebook/presto/operator/scalar/TestStringFunctions.java

```diff
@@ -509,6 +672,16 @@ public void testLower()
     }
 
     @Test
+    public void testCharLower()
+    {
+        assertFunction("LOWER(CAST('' AS CHAR(10)))", createCharType(10), "");
```

Comment:
remove `lowerByCodePoint` and `upperByCodePoint` from expected value and inline correct one.